src/witan/models/dem/ccm/mort/mortality_mvp.clj

```diff
@@ -8,6 +8,10 @@
             [incanter.core :as i]))
 
 (defn create-popn-at-risk
+  "Takes datasets with population, deaths, and births. Returns a new
+  dataset with population at risk in the :popn column. The population
+  at risk is the births data joined with a subset of the population
+  dataset that has been aged on."
```

Comment:
move line 21(query dataset) to below line 18 (inc year) - reduces amount of code to do remaining fns on - see if it breaks test!